nextflow_schema.json

```diff
@@ -132,6 +132,13 @@
                 "binaryref": {
                     "type": "string",
                     "description": "Whether to generate a binary reference file to be used with GLIMPSE2"
+                },
+                "chunk_model": {
+                    "type": "string",
+                    "description": "Model type to use for GLIMPSE2_CHUNK",
+                    "enum": ["recursive", "sequential", "uniform-number-variants"],
```

Comment:
There should be a test assessing each of these modes (at least the new ones, sequential and uniform-number-variants).

Reply:
I will create them in the subworkflows.

Reply:
Great!